src/views/faq.pug

```diff
@@ -160,6 +160,11 @@ block content
       li #[code *.while-at.school]
       li #[code *.while-at.work]
 
+    p.
```

Comment:
I don't like this wording.

```pug
      Some of these domains may be broken when you go to use them (they
      #[em are] donated, after all). A website called
      #[a(href='https://owostat.us' ref='nofollow' target='_blank') owostat.us]
      has been created by the community to mirror the list on this website with
      green or red colors to signal whether the domain is working with owo or
      not.
```
Even this wording isn't that great to be honest.